services/auth-service/src/main/java/com/sourcefuse/jarc/services/authservice/controller/PingController.java

```diff
@@ -0,0 +1,15 @@
+package com.sourcefuse.jarc.services.authservice.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/")
+public class PingController {
```

Comment:
remove this and use https://reflectoring.io/spring-boot-health-check/

Reply:
done